src/main.cpp

```diff
@@ -353,9 +353,16 @@ namespace irods::plugin::rule_engine::audit_amqp
 		std::map<std::string, std::size_t> arg_type_map;
 
 		ruleExecInfo_t* rei = nullptr;
-		irods::error err = _eff_hdlr("unsafe_ms_ctx", &rei);
-		if (!err.ok()) {
-			return err;
+		if (const auto err = _eff_hdlr("unsafe_ms_ctx", &rei); !err.ok()) {
+			// clang-format off
+			log_re::trace({
+				{"rule_engine_plugin", rule_engine_name},
+				{"log_message", "could not get rule execution context (REI)"},
+				{"rule_name", _rn},
+				{"error_result", err.result()},
```

Comment:
Extra comma

Reply:
> Couple of things that apparently do not matter for compilation(?)

That's correct. I'm guessing that is a feature of the language. Other languages allow that too.

Will remove.

Reply:
Removed.